deploy/helm/microfunctions/Chart.yaml

```diff
@@ -16,4 +16,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: High Performance Serverless system
 name: MicroFunctions
-version: auto
+version: 0.8.8
```

Comment:
Does this mean we'll have to update this value every time we make a release?

Reply:
Thx for your comment. The version is always being set in the Makefile build script. "auto" wasn't a real version (helm would complain that it's not a valid chart), I think it has some merit to have a working helm chart, but at the same time I couldn't think of what to set it to.
- I used the latest release version, so when people use the helm chart folder (deploy/helm/microfunctions) instead of the Makefile-generated release Helm chart package (MicroFunctions-x.y.z.tgz), then they would deploy our latest release.
- we could also set it to 0.0.1, so it would be an obvious configuration error if it appears somewhere
- we can revert it to a bogus version (auto) or remove it altogether, which makes the folder not work as a helm chart.

what would you suggest?

Reply:
FYI, here's the line where the release helm chart package gets its actual version: https://github.com/knix-microfunctions/knix/blob/b9f45c3d57056707eb22f913748ecf251612e9af/deploy/helm/Makefile#L28

Reply:
The old Makefile used to set this to 'auto' before packaging. How about we utilize the same idea to set the actual version in the Makefile?

This commit shows the difference: https://github.com/knix-microfunctions/knix/pull/104/commits/18da1e12af99bfc5b09307f9b858421e75f9d06b

Instead of sed 's/^version:.*/version: auto/' -i microfunctions/Chart.yaml, we can set it to sed 's/^version:.*/version: $(VERSION)/' -i microfunctions/Chart.yaml. In fact, that is shown on the Line26 of the same commit.

Reply:
I guess it's not really important. So it doesn't matter.

Reply:
Good point, I've added to _also_ update the version. It'll appear as a modified file in git but then be in sync with the generated package.